rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -3103,7 +3103,7 @@ pub mod settle_neurons_fund_participation_request {
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Principals {
-    #[prost(message, repeated, tag = "7")]
```

Comment:
Is anything wrong with 7?

Reply:
Not that I know of, but 1 just seems cleaner for the only element